sherpa/astro/ui/utils.py

```diff
@@ -5105,16 +5105,19 @@ def save_data(self, id, filename=None, bkg_id=None, ascii=True, clobber=False):
             d = self.get_bkg(id, bkg_id)
 
         try:
-            sherpa.astro.io.write_pha(filename, d, ascii, clobber)
+            sherpa.astro.io.write_pha(filename, d, ascii=ascii,
+                                      clobber=clobber)
         except IOErr:
             try:
-                sherpa.astro.io.write_image(filename, d, ascii, clobber)
+                sherpa.astro.io.write_image(filename, d, ascii=ascii,
+                                            clobber=clobber)
             except IOErr:
                 try:
-                    sherpa.astro.io.write_table(filename, d, ascii, clobber)
+                    sherpa.astro.io.write_table(filename, d, ascii=ascii,
```

Comment:
The explicit settings of ascii and clobber here are unnecessary functionally as the parameter list matches the API in astro/io/__init__.py.